docs/source/getting_started/learning_by_example.rst

```diff
@@ -22,129 +26,98 @@ manim. For instance, take ``SquareToCircle``. Copy the code given below to a new
+  Other frequently used flags are:
+
+    * ``-l`` for rendering video in lower resolution (which renders faster)
+    * ``-s`` to show the last frame of the video.
+
+  Run ``manim -h`` all the available flags (``python -m manim -h`` if you installed it to a venv)
+
+
+Let's step through each line of ``SquareToCircle``
+
+Each :class:`~scene.scene.Scene` in manim is self-contained. That means everything
+you created under this scene does not exist outside the class.
+
+:meth:`~scene.scene.Scene.construct` specifies what is displayed on the screen
+when the :class:`~scene.scene.Scene` is rendered to video.
+
+``Circle()`` and ``Square`` :class:`~mobject.geometry.Circle` and
+:class:`~mobject.geometry.Square`, respectively.
 
 .. code-block:: python
    :linenos:
-   :lineno-start: 3
+   :lineno-start: 5
 
    circle = Circle()
    square = Square()
 
-Lines 5, 6, and 7 apply various modifications to the mobjects before animating
-them.  The call to :meth:`~mobject.mobject.Mobject.flip` on line 5 flips the
-:class:`~mobject.geometry.Square` across the RIGHT vector.  This is equivalent
-to a refection across the x-axis. Then the call to
-:meth:`~mobject.mobject.Mobject.rotate` on line 6 rotates the
-:class:`~mobject.geometry.Square` 3/8ths of a full rotation counterclockwise.
-Finally, the call to :meth:`~mobject.mobject.Mobject.set_fill` on line 7 sets
-the fill color for the :class:`~mobject.geometry.Circle` to pink, and its
-opacity to 0.5.
+Both of these subclass :class:`~mobject.mobject.Mobject`, the base class for objects in manim. Note
+that instantiating a :class:`~mobject.mobject.Mobject` does not add it to the
+:class:`~scene.scene.Scene`, so you wouldn't see anything if you were to render
+the :class:`~scene.scene.Scene` at this point.
 
 .. code-block:: python
    :linenos:
-   :lineno-start: 5
+   :lineno-start: 7
 
    square.flip(RIGHT)
    square.rotate(-3 * TAU / 8)
    circle.set_fill(PINK, opacity=0.5)
 
-Line 9 is the first to generate video.
-:class:`~animation.creation.ShowCreation`,
-:class:`~animation.transform.Transform`, and
-:class:`~animation.creation.FadeOut` are
-:class:`~animation.animation.Animation` instances. Each
-:class:`~animation.animation.Animation` takes one or more
-:class:`~mobject.mobject.Mobject` instances as arguments, which it animates
-when passed to :meth:`~scene.scene.Scene.play`. This is how video is typically
-created in manim. :class:`~mobject.mobject.Mobject` instances are automatically
-added to the :class:`~scene.scene.Scene` when they are animated. You can add a
-:class:`~mobject.mobject.Mobject` to the :class:`~scene.scene.Scene` manually
-by passing it as an argument to :meth:`~scene.scene.Scene.add`.
+``flip()`` ``rotate()`` ``set_fill()`` apply various modifications to the mobjects before animating
+them.  The call to :meth:`~mobject.mobject.Mobject.flip` flips the
+:class:`~mobject.geometry.Square` across the RIGHT vector.  This is equivalent
```

Comment:
I don't know the point of this. Square will show no changes when flip across right.